docs/blog/_posts/2019-01-21-12th-dotty-milestone-release.md

```diff
@@ -35,19 +35,41 @@ This is our 12th scheduled release according to our
 ### Extension Methods
 
 We are excited to announce that extension methods are now offered through dedicated language support!
-Extension methods allow one to add methods to a type after the type is defined and up until now they were encoded through the powerful mechanism of implicits. 
-Previously, one had to place the desired method in the trait with the receiver object to extend, as part of the parameter list. 
-The infix-ness of the method was being restored by manually writing the implicit class, resolving `implicitly` that extended object. 
-
-Now, in Dotty we can express an extension method by writing the method and prepending its name with the type to be extended. 
+Extension methods allow one to add methods to a type after the type is defined.
+This is done by writing a method with a parameter for the type to be extended
+_on the right-hand side_ of the method name:
 
 ```scala
 case class Circle(x: Double, y: Double, radius: Double)
 
 def (c: Circle) circumference: Double = c.radius * math.Pi * 2
 ```
 
-Read the [relevant documentation](https://dotty.epfl.ch/docs/reference/other-new-features/extension-methods.html) about generic extension methods, higher-kinded extension methods and more.
+Extension methods were previously encoded in a rather roundabout way via the implicit class pattern.
+Such encoding required a lot of boilerplate, especially when defining type classes.
+In Dotty, this is no longer the case,
+and type classes with infix syntax become very straightforward to define!
+For example, consider:

```

Comment:
I think we dive in the topic of extension methods & type classes directly by skipping generic extension methods. Yes, it has been less straightforward before, but extension methods are a general language feature (others coming from other ecosystems do not use them for type classes only). This is why I left more elaborate examples out.

Reply:
By _elaborate_, you mean that it has a type parameter? I think most Scala users (even beginners) won't have a problem understanding this code snippet. But of course I can change `ListSemigroup` to `StringSemigroup` for example, if that makes it simpler.